run_checks.php

```diff
@@ -88,6 +88,17 @@
     exit(1);
 })->wait();
 
+$webhookSecret = "secret";
+$webhookData = "data";
+$webhookHeader = "v1=1b2c16b75bd2a870c114153ccda5bcfca63314bc722fa160d690de133ccbb9db";
+$isValidWebhookSign = \Fingerprint\ServerAPI\Webhook\WebhookVerifier::IsValidWebhookSignature($webhookHeader, $webhookData, $webhookSecret);
```

Comment:
Let's apply `use $NAMESPACE` to all identifiers in this file to make them more readable